### PR TITLE
[Jupyter] Reusing pipeline name in a different project should not error

### PR DESCRIPTION
**Pull request by bbonenfant**

### CODE REVIEW

File: jupyter-extension/jupyterlab_pachyderm/pps_client.py [hunk 1]
```diff
@@ -95,7 +94,11 @@ async def create(self, path: str, body: dict):
             )
 
         companion_repo = f"{config.pipeline.name}__context"
-        if companion_repo not in (item.repo.name for item in client.list_repo()):
+        for item in client.list_repo():
+            same_project = config.pipeline.project.name == item.repo.project.name
+            if (companion_repo == item.repo.name) and same_project:
+                break
+        else:
```
Comment (kevyang) [inline jupyter-extension/jupyterlab_pachyderm/pps_client.py:101]: indentation seems off here

Comment (bbonenfant) [reply jupyter-extension/jupyterlab_pachyderm/pps_client.py:101]: This is actually proper syntax for the under-utilized for/else block: https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops

**Comment by chainlink** [comment]
How does this relate to https://github.com/pachyderm/pachyderm/pull/8806 ?

**Review by chainlink — APPROVED** [review]

**Comment by bbonenfant** [comment]
> How does this relate to #8806 ?

The two are unrelated.

**Pull request merged**